src/gui/src/displayControls.cpp

```diff
@@ -1731,4 +1740,47 @@ void DisplayControls::setOnlyVisibleLayers(const std::set<const odb::dbTechLayer
   }
 }
 
+void DisplayControls::postReadLiberty()
+{
+  checkLiberty();
+}
+
+void DisplayControls::checkLiberty()
+{
+  bool enable = true;
+
+  if (sta_ == nullptr) {
+    enable = false;
+  } else {
+    auto* network = sta_->getDbNetwork();
+    if (network->defaultLibertyLibrary() == nullptr) {
+      enable = false;
+    }
+  }
```

Comment:
Unfortunately the postReadLiberty callback comes before the setting of the default library so the enabling doesn't work.  I think the callback itself will have to serve as sufficient evidence of a liberty.  You could have the call from postReadLiberty tell checkLiberty to enable without checking through an argument.

Reply:
Thanks for catching that, my testcase had memories, so I never just loaded one. Should have been addressed in f59b702